packages/build-info/package.json

```diff
@@ -3,9 +3,11 @@
   "version": "5.1.1",
   "description": "Build info utility",
   "type": "module",
-  "exports": "./lib/main.js",
-  "main": "./lib/main.js",
-  "types": "./lib/main.d.ts",
+  "exports": {
+    ".": "./lib/index.js"
```

Comment:
restrict subpath imports (this should help to not introduce breaking changes that easily)

Only functions that are exported from `lib/index.js` are public api surface